src/components/index.ts

```diff
@@ -23,3 +23,4 @@ export * from './ToggleGroup';
 export * from './Tooltip';
 export { Wizard } from './Wizard';
 export * from './Checkbox';
+export * from './Status';
```

Comment:
nit: could we keep these in alphabetical order for consistency please 🙂

Reply:
Updated